ietf/templates/group/group_about.html

```diff
@@ -401,8 +404,38 @@ <h2 class="mt-3">
             </p>
         {% endif %}
     {% endif %}
+    {% if subgroups %}
+        <h2 class="mt-4">
+            Subgroups under {{ group.acronym|upper }}
```

Comment:
Normal people would expect to just see "Groups under ART" not "Subgroups under ART"

Reply:
And consider "Group in ART" rather then "under"